clisops/core/subset.py

```diff
@@ -788,10 +789,14 @@ def shape_bbox_indexer(ds, poly):
     # Find indices nearest the rectangle' corners
     # Note that the nearest indices might be inside the shape, so we'll need to add a *halo* around those indices.
     if rectilinear:
-        native_ind, _ = xarray.core.coordinates.remap_label_indexers(
-            ds, ind, method="nearest"
-        )
-
+        if version.parse(xarray.__version__) < version.Version("2022.06.0"):
```

Comment:
is backward compatibility necessary?

Reply:
I could add a deprecation warning? Remove in the following version?